docs/collecting-data/collecting-from-own-applications/mobile-trackers/configuring-how-events-are-sent/index.md

```diff
@@ -199,38 +199,106 @@ EmitterConfiguration emitterConfiguration = new EmitterConfiguration()
+In case the default emit range (25) is used, the tracker will make 4 requests serially, one after the other, each with 25 events.
 
 If the event store is empty when the tracker tries to send events - because another thread has just sent them - the thread sleeps for 5 seconds before trying again. If this happens 5 times in a row in the same thread, event sending will be paused for the whole tracker. It is restarted when a new event arrives.
 
-Configure the batch size like this:
+Configure the batch size and emit range like this:
+
+<Tabs groupId="platform" queryString>
+  <TabItem value="ios" label="iOS" default>
+
+```swift
+let emitterConfig = EmitterConfiguration()
+      .bufferOption(BufferOption.single)
+      .emitRange(25)
+```
+
+  </TabItem>
+  <TabItem value="android" label="Android (Kotlin)">
+
+```kotlin
+val emitterConfiguration = EmitterConfiguration()
+    .bufferOption(BufferOption.Single)
+    .emitRange(25)
+```
+
+  </TabItem>
+  <TabItem value="android-java" label="Android (Java)">
+
+```java
+EmitterConfiguration emitterConfiguration = new EmitterConfiguration()
+      .bufferOption(BufferOption.Single)
+      .emitRange(25);
+```
+
+  </TabItem>
+</Tabs>
+
+:::note Behaviour before version 6.0.0 of the tracker
+Before version 6 of the iOS and Android tracker, the `bufferOption` and `emitRange` had a slightly different meaning.
+Events were sent right after they were tracked regardless of the `bufferOption` used.
+The `bufferOption` was used to specify the maximum number of events per request and the `emitRange` specified the maximum number of total events to make in parallel requests at once.
+For instance, if there were 100 events in the event store and `bufferOption` was set to 10 (called `defaultGroup` previously) and `emitRange` to 50, the tracker would make 5 parallel requests to the collector with 10 events each. After that it would make another 10 parallel requests to the collector with 10 events each.
+:::
+
+## Automatic clean up of the event store
+
+:::note Not available before v6
+This feature was introduced in version 6.0.0 of the iOS and Android trackers.
+:::
+
+Under some situations events may keep accumulating in the event store without the tracker being able to send them at all.
+For instance, this may happen in case the user has an ad blocker installed or in case they permanently use the app offline.
+
+If events accumulated in the event store without any limits, the size of the event store would keep getting bigger, potentially causing performance issues or app crashes.
+To prevent this, the tracker automatically removes old events from the event store.
+It removes old events based on two criteria that are configurable using `EmitterConfiguration`:
+
+1. Maximum event store size (`maxEventStoreSize`) – in case the number of events surpasses this threshold, oldest events will be removed until the number of events is under the threshold. Defaults to 1000.
```

Comment:
```suggestion
1. Maximum event store size (`maxEventStoreSize`) – in case the number of events surpasses this threshold, the oldest events will be removed until the number of events is under the threshold. Defaults to 1000.
```